.github/workflows/e2e-tests.yml

```diff
@@ -32,51 +26,43 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        target:
-          - e2e
-          - sync
-    
+        target: ["./e2e/validity/...", "./e2e/faultproof/..."]
+
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v4
-      with:
-        submodules: recursive
-      
-    - name: Setup Rust
-      run: |
-        rustup toolchain install stable --profile minimal
-        rustup component add rustfmt clippy
-        
-    - name: Install Foundry
-      uses: foundry-rs/foundry-toolchain@v1
-      with:
-        version: nightly-d592b3e0f142d694c3be539702704a4a73238773
-        
-    - name: Cache Rust dependencies
-      uses: Swatinem/rust-cache@v2
-      with:
-        workspaces: |
-          .
-          bindings
+      - name: Checkout sources
+        uses: actions/checkout@v5
+        with:
+          submodules: true
+
+      - name: Setup Rust
+        run: |
+          rustup toolchain install stable --profile minimal
+
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly-d592b3e0f142d694c3be539702704a4a73238773
+          
+      - name: Cache Rust dependencies
+        uses: Swatinem/rust-cache@v2
+        with:
+          workspaces: |
+            .
+            bindings
+
+      - name: Install just
+        uses: extractions/setup-just@v3
 
-    - name: Install just
-      uses: extractions/setup-just@v3
-        
-    - name: Run E2E tests
-      run: just e2e-tests ${{ matrix.target }}
-      env:
-        # TODO(fakedev9999): Remove L1 RPCs since we are using fresh Anvil chain
-        L1_RPC: ${{ secrets.L1_RPC }}
-        L1_BEACON_RPC: ${{ secrets.L1_BEACON_RPC }}
-        L2_RPC: ${{ secrets.L2_RPC }}
-        L2_NODE_RPC: ${{ secrets.L2_NODE_RPC }}
+      - name: Setup Go 1.23.8
+        uses: actions/setup-go@v5
+        with:
+          go-version: '1.23.8'
+          cache-dependency-path: |
+            tests/go.sum
+            tests/optimism/go.sum
 
-    - name: Upload test logs on failure
```

Comment:
Did you remove this on purpose? Maybe we run go tests with -v flag so we're fine without it?